src/Assets/Javascript/buildHTML.js

```diff
@@ -0,0 +1,228 @@
+    finalStructure.push([mainBoxDiv, likeArea, 'd-flex justify-content-center']);
+    finalStructure.push([likeArea, catName, null, array[i].breeds[0].name]);
+    if (getLocalLikesPrev[array[i].breeds[0].id]) {
+      finalStructure.push([likeArea, likeHeart, 'bi bi-heart-fill ms-4']);
+    } else {
+      finalStructure.push([likeArea, likeHeart, 'bi bi-heart ms-4']);
+    }
+    finalStructure.push([mainBoxDiv, likesCount, 'likesCount', `${likesCounter} likes`, `${likesCounter}`]);
+    finalStructure.push([mainBoxDiv, commentBtn, 'btn', 'Comments']); // don't move this
+  }
+  htmlBuilder(finalStructure);
+};
+
+export const buildModals = (array, commentsArray) => {
+  const finalStructure = [];
+  for (let i = 0; i < array.length; i += 1) {
+    const mainBoxDiv = document.getElementById(`mainBoxDiv${i}`);
+    const popUpCointainer = document.createElement('div');
+    const modalDialog = document.createElement('div');
+    const modalContent = document.createElement('div');
+    const modalHeader = document.createElement('div');
+    const modalPicture = document.createElement('img');
+    const modalTitle = document.createElement('h5');
+    const btnClose = document.createElement('button');
+    const modalBody = document.createElement('div');
+    const div = document.createElement('div');
+    const div2 = document.createElement('div');
+    const div3 = document.createElement('div');
+    const div4 = document.createElement('div');
+    const data1 = document.createElement('span');
+    const data2 = document.createElement('span');
+    const div5 = document.createElement('div');
+    const div6 = document.createElement('div');
+    const div7 = document.createElement('div');
+    const data3 = document.createElement('span');
+    const data4 = document.createElement('a');
+    const div8 = document.createElement('div');
+    const commentTitle = document.createElement('h5');
+    const nameInput = document.createElement('input');
+    nameInput.setAttribute('placeholder', 'Your name');
+    const commentInput = document.createElement('textarea');
+    commentInput.setAttribute('placeholder', 'Your insights');
+    data4.setAttribute('href', array[i].breeds[0].wikipedia_url);
+    const commentBtn = document.createElement('button');
+
+    commentBtn.addEventListener(('click'), () => {
+      const commentContainer = document.getElementById(`comContainertId${i}`);
+      postComment(involvementApi, array[i].breeds[0].id, nameInput.value, commentInput.value);
+      const finalInnerStructure = [];
+      let commentsArray2 = [];
+      const breedCats = ['abys', 'aege', 'abob', 'amau', 'amis', 'bamb', 'bslo', 'cspa', 'beng'];
+      const commentsApi = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/GTdCo4dMv7OdJ4VT5RJ0/comments?item_id=';
+      setTimeout(() => {
+        commentsArray2 = saveComments(commentsApi, breedCats);
+      }, 1500);
+      setTimeout(() => {
```

Comment:
* like the previous reviewer stated when a comment is added the comments popup window isn't updated with stored comments kindly fix this issue 🙏 
* Users shouldn't have to reload the page for comments to be visible, they should be able to see updates immediately